client/src/scss/section.scss

```diff
@@ -20,6 +20,7 @@
     flex-direction: row;
     align-items: center;
     margin-bottom: 34px;
+    padding-left: 8px;
```

Comment:
In #188 requirements are to add padding to "Latest Message" on Landing page.Adding padding left to .section--header class could reflect on all other pages where this class is used. Instead, consider using any of padding-l util classes.